src/plugins/intel_gpu/tests/functional/subgraph_tests/loop.cpp

```diff
@@ -401,38 +407,38 @@ class DynamicShapeLoopDynamicInputTest : public testing::WithParamInterface<Dyna
         // It should be updated during iteration
         auto start_add = cond_input_create(model_type, inputShape, start_value);
         start_add->set_friendly_name("start_add");
-        auto start_mul = cond_input_create(model_type, inputShape, 1);
-        start_mul->set_friendly_name("start_mul");
+        auto start_add2 = cond_input_create(model_type, inputShape, 1);
+        start_add2->set_friendly_name("start_add2");
         auto count = cond_input_create(ov::element::i64, scalarShape, max_iter_num, static_iter_num);
         count->set_friendly_name("count");
         auto skip  = cond_input_create(ov::element::boolean, scalarShape, true, static_continue_cond);
         skip->set_friendly_name("skip");
-        auto init_const = cond_input_create(model_type, constant_shapes.first, 1);
+        auto init_const = cond_input_create(model_type, constant_shapes.first, 1, freeze_input);
         init_const->set_friendly_name("init_const");
 
         auto b_indx = std::make_shared<ov::op::v0::Parameter>(ov::element::i64, ov::Shape{});
         b_indx->set_friendly_name("body_index");
         auto b_data_add = std::make_shared<ov::op::v0::Parameter>(model_type, inputShape);
         b_data_add->set_friendly_name("b_data_add");
-        auto b_data_mul = std::make_shared<ov::op::v0::Parameter>(model_type, inputShape);
-        b_data_mul->set_friendly_name("b_data_mul");
+        auto b_data_add2 = std::make_shared<ov::op::v0::Parameter>(model_type, inputShape);
+        b_data_add2->set_friendly_name("b_data_add2");
         auto b_data_broadcast = std::make_shared<ov::op::v0::Parameter>(model_type, constant_shapes.first);
         b_data_broadcast->set_friendly_name("b_data_broadcast");
         auto b_indx_cast = std::make_shared<ov::op::v0::Convert>(b_indx, model_type);
         b_indx_cast->set_friendly_name("body_index_cast");
         auto b_add  = std::make_shared<ov::op::v1::Add>(b_data_add, b_indx_cast);
         b_add->set_friendly_name("body_add");
-        auto b_mul  = std::make_shared<ov::op::v1::Multiply>(b_data_mul, b_indx_cast);
-        b_mul->set_friendly_name("body_mul");
-        auto b_shapeof1 = std::make_shared<ov::op::v3::ShapeOf>(b_data_mul);
+        auto b_add2  = std::make_shared<ov::op::v1::Add>(b_data_add2, b_indx_cast);
```

Comment:
Why changed the operation?

Reply:
This operation will give a broadcast operation a different input shape in various iterations of the loop primitive and ensure that the input shape in the second iteration is different from that in the first iteration( which goes through the back edge).